src/data/employees.ts

```diff
@@ -584,6 +594,52 @@ const shreyaBhattarai: Employee = {
     linkedInLink: 'https://www.linkedin.com/in/shreyabha/',
 };
 
+const sushilTiwari: Employee = {
+    id: 'tc-072',
+    name: 'Sushil Tiwari',
+    position: 'Backend Developer',
+    description: 'Waasssssssssup Bro?',
+    funnyDescription: 'Waasssssssssup Bro?',
```

Comment:
description and funnyDescription cannot be the same. It is something the team members decide